internal/snmp/translate_test.go

```diff
@@ -18,23 +18,23 @@ func TestTrapLookup(t *testing.T) {
 			name: "Known trap OID",
 			oid:  ".1.3.6.1.6.3.1.1.5.1",
 			expected: MibEntry{
-				MibName: "SNMPv2-MIB",
+				MibName: "TGTEST-MIB",
 				OidText: "coldStart",
 			},
 		},
 		{
 			name: "Known trap value OID",
 			oid:  ".1.3.6.1.2.1.1.3.0",
 			expected: MibEntry{
-				MibName: "SNMPv2-MIB",
-				OidText: "sysUpTime.0",
+				MibName: "TGTEST-MIB",
+				OidText: "sysUpTimeInstance",
```

Comment:
@reimda This test was specifically crafted to have a test with an `OidText` having 1 additional part (`.0`). The next test with enterprise tests if it also works with multiple additional parts. The previous test already tests an fully translatable OID.

Reply:
I understand that oids ending with .0 often or usually refer to an instance of a scalar object. Most of the time there is no name defined in the mib for the instance, but in this case, the SNMPv2 authors did include a name "sysUpTimeInstance".

I think it's a good idea to include a test for an instance of a scalar object that doesn't have a name in the mib. In that case we will need to choose a different oid than sysUpTime, and one that doesn't have a name (like xyzInstance) in the mib.

One way to do this would be to remove sysUpTimeInstance from the mib and use SysUpTime.0 like you had, but I think this would be confusing to people who are familiar with these common oids. Let's add a new oid scalar object that has a clearly fake name like "scalarObject" and use it in the test.

Does this sound good to you? Could you open a new PR to do this?

Reply:
I would have preferred `sysUptime.0` but I understand your reasoning and agree to add a new object for that.